server/src/test/java/org/elasticsearch/transport/RemoteClusterServiceTests.java

```diff
@@ -110,6 +116,37 @@ public void testRemoteClusterSeedSetting() {
         assertEquals("failed to parse port", e.getMessage());
     }
 
+    public void testRemoteClusterOptionalCredentialSettingValues() {
```

Comment:
What is this test testing? I don't see any assertions.

Reply:
I wrapped the last line with a try/catch. The catch has a fail message, instead of letting the exception bubble up JUnit. I tweaked the name of the test to hopefully clarify the test too.